browser-sign-in/app/src/main/java/com/okta/android/samples/browser_sign_in/views/UserDashboardFragment.kt

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2022-Present Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.okta.android.samples.browser_sign_in.views
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import androidx.navigation.fragment.findNavController
+import com.okta.android.samples.browser_sign_in.R
+import com.okta.android.samples.browser_sign_in.data.*
+import com.okta.android.samples.browser_sign_in.databinding.FragmentUserDashboardBinding
+
+class UserDashboardFragment : Fragment() {
+    private val viewModel: UserDashboardViewModel by viewModels()
+    private lateinit var binding: FragmentUserDashboardBinding
```

Comment:
Same potential leak mentioned in the LoginFragment exists here as well.

Reply:
Whoops! I will fix that